xml/security_pcidss.xml

```diff
@@ -7,272 +7,1251 @@
+        TLS connection. The authentication is done with a public
+        x.509 certificate that proves to a certain level that the server is
+        the right endpoint the customer is looking for.
+       </para>
+       <para>
+        &productname; comes with a set of services and tools that allow
+        protected HTTPS connections. For example, this can be directly done
+        with the Apache HTTP Server or via <command>stunnel</command> that
+        functions as a proxy to offer TLS encryption functionality.
+       </para>
+       <para>
+        IPsec or other VPN technologies can be used for securing the
+        connection between network segments that are connected via a public
+        network. Such connections can also be secured with a public x.509
+        certificate. For internal usage it would be possible to use a private
+        Certificate Authority (CA) to sign x.509 certificates and to keep
+        track on trusted keys.
+       </para>
+       <para>
+        In &productname; this can be directly established with
+        <phrase role="productname">strongSwan</phrase> that is a IPsec-based
+        VPN solution or with OpenVPN that is using a custom security protocol.
+       </para>
+       <para>
+        Administration access to the OS level is usually done via SSH.
+        Configuration and authentication can be tightened to meet the needed
+        security. This was partially stated before inside the requirements 1
+        and 2.
+       </para>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+   </sect3>
+  </sect2>
+  <sect2>
+   <title>Maintain a Vulnerability Management Program</title>
+   <para/>
+   <sect3>
+    <title>Requirement 5: Protect All Systems Against Malware and Regularly Update Anti-Virus Software or Programs</title>
+    <para>
+     For &pcidssa; compliance it is necessary to protect against malicious
+     software threats. Commercial anti-virus software is available from the
+     major anti-virus software vendors and can be integrated into the Linux
+     environment. &productname; itself comes with the open source anti-virus engine
+     &clamav;.
+    </para>
+    <para>
+     The question here is which functionality and performance is needed.
+     &clamav; has a limited set of scanning capabilities and limited
+     performance compared to commercial products. So the expectations need to
+     be clear that &clamav; only gives only a simple protection level.
+    </para>
+    <para>
+     On the other hand &clamav; is directly shipped with &productname; and can be
+     included during server installation. This makes it easy to fulfill the
+     details of this requirement, but the drawbacks compared to commercial
+     products need to be clearly understood.
+    </para>
+   </sect3>
+   <sect3>
+    <title>Requirement 6: Develop and maintain secure systems and applications</title>
+    <para>
+     A powerful instrument in &productname; is the tool
```

Comment:
A powerful instrument in &productname; is the tool <command>zypper</command> that is an interface to

<command>zypper</command>  is a powerful tool that provides interface to